gst/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -40,6 +67,71 @@ struct _Tflite_data
 };
 typedef struct _Tflite_data tflite_data;
 
+/**
+  * @brief parse custom options for tensorflow lite
+  * @param filter : tensor_filter instance
+  * @param id : Option ID
+  * @return 0 or 1 if successfully parsed.
+  *        -1 if parsing a value of a given option failed.
+  */
+static int
+tflite_get_parser_boolean (const GstTensorFilter * filter, int id)
+{
+  int ret = -1;
+
+  if (filter->prop.custom_properties == NULL)
+    return ret;
+
+  if (id < 0 || id >= TFLITE_OPTION_MAX) {
+    GST_WARNING ("Invalid option id. (%d)\n", id);
+    return ret;
+  }
+
+  custom_options = g_strdup (filter->prop.custom_properties);
+  if (custom_options == NULL) {
+    GST_WARNING ("failed to dump custom property string\n");
+    return ret;
+  }
+
+  char *str = strtok (custom_options, tokens);
+  while (str != NULL) {
+    if (strcmp (str, tflite_options[id]) == 0) {
+      str = strtok (NULL, tokens);
+      break;
+    }
```

Comment:
This algorithm loops the whole custom_properties for each single option.
Please loop once and parse all custom properties.